content/en/getting-started-overview.md

```diff
@@ -0,0 +1,127 @@
+  that allows you to install, update, and remove Tekton projects on your Kubernetes cluster.
+
+
+## How do I work with Tekton?
+
+To install Tekton, you need a Kubernetes cluster running a
+[version of Kubernetes specified for the current Tekton release](https://github.com/tektoncd/pipeline/#want-to-start-using-pipelines).
+Once installed, you can interact with Tekton using one of the following:
+
+- **[The tkn CLI](https://github.com/tektoncd/cli/blob/master/README.md)**, also known as the Tekton CLI, is the preferred command-line method
+  for interacting with Tekton. `tkn` provides a quick and streamlined experience, including high-level commands and color coding. To use it,
+  you only need to be familiar with Tekton.
+- **[The kubectl CLI](https://kubernetes.io/docs/reference/kubectl/overview/)**, also known as the Kubernetes CLI, provides substantially more
+  granularity for controlling Tekton at the expense of higher complexity. Interacting with Tekton via kubectl is typically reserved for debugging
+  your pipelines and troubleshooting your builds.
+- **[The Tekton APIs](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/)**, currently
+  available for [Pipelines](https://pkg.go.dev/github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1?tab=doc) and
+  [Triggers](https://pkg.go.dev/github.com/tektoncd/triggers@v0.5.0/pkg/apis/triggers/v1alpha1?tab=doc), allow you to programmatically interact
+  with Tekton components. This is typically reserved for highly customized CI/CD systems. In most scenarios, `tkn` and `kubectl` are the preferred 
+  methods of controlling Tekton.
+
+We also recommend having the following items configured on your Kubernetes cluster:
+
+- Persistent volume claims for specifying inputs and outputs.
+- Permissions appropriate to your environment and business needs.
+- Storage for building and pushing images (if applicable).
+
+## What can I do with Tekton?
+
+Tekton introduces the concept of `Tasks`, which specify the workloads you want to run:
+
+- `Task` - defines a series of ordered `Steps`, and each `Step` invokes a specific build tool on a specific set of inputs and
+  produces a specific set of outputs, which can be used as inputs in the next `Step`. 
+
+- `Pipeline` - defines a series of ordered `Tasks`, and just like `Steps` in a `Task`, a `Task` in a `Pipeline` can use the output
+  of a previously executed `Task` as its input.
+
+- `TaskRun` - instantiates a specific `Task` to execute on a particular set of inputs and produce a particular set of outputs. In
+  other words, the `Task` tells Tekton what to do, and a `TaskRun` tells Tekton what to do it **on**, as well as any additional details
+  on how to exactly do it, such as build flags.
+
+- `PipelineRun` - instantiates a specific `Pipeline` to execute on a particular set of inputs and produce a particular set of outputs
+  to particular destinations.
```

Comment:
would it make sense to link these to pages that go into more detail about these types?

Reply:
I think the idea here is to be high-level and just introduce the concepts, then in the "I want to learn more" section, we link to the Pipelines Overview which explains these entities more and links to the full docs. I thought it would be best to have the reader ingest this page as a whole without jumping away to super-dense technical docs right away.

Reply:
i dont feel strongly enough to stop us from merging, but i do think there's a good chance someone will jump around the page and it might be useful to give them the links they need here (e.g. maybe they never make it to "i want to learn more")

Reply:
i dont feel strongly enough to stop us from merging, but i do think there's a good chance someone will jump around the page and it might be useful to give them the links they need here (e.g. maybe they never make it to "i want to learn more")

Reply:
That's a good point - links added!